src/mobile/ios.mm

```diff
@@ -183,6 +198,28 @@ -           (BOOL) application: (UIApplication*) application
   viewController.view.frame = appFrame;
   self.window.rootViewController = viewController;
 
+  NSNotificationCenter* ns = [NSNotificationCenter defaultCenter];
+  [ns addObserver: self selector: @selector(keyboardDidShow) name: UIKeyboardDidShowNotification object: nil];
+  [ns addObserver: self selector: @selector(keyboardDidHide) name: UIKeyboardDidHideNotification object: nil];
+  [ns addObserver: self selector: @selector(keyboardWillShow) name: UIKeyboardWillShowNotification object: nil];
+  [ns addObserver: self selector: @selector(keyboardWillHide) name: UIKeyboardWillHideNotification object: nil];
+  [ns addObserver: self selector: @selector(keyboardWillChange:) name: UIKeyboardWillChangeFrameNotification object: nil];
+
+  [self listenForKeyDown];
+  [self setUpWebView];
+  [self.window makeKeyAndVisible];
+
+  return YES;
+}
+
+- (void)setUpWebView
+{
+  [self.content removeAllUserScripts];
+  self.webview.navigationDelegate = nil;
+  self.webview.scrollView.delegate = nil;
+  [self.webview stopLoading];
+  [self.webview removeFromSuperview];
```

Comment:
Not sure why, but this isn't enough for the previous web view to be released. It still shows up in Safari inspector's `Develop` menu.